### DISCUSSION TITLE
opensource without politics

### DISCUSSION THREAD

**Discussion by mikesrus** [arendst/Tasmota#15023] [General]
I see some limitations in future versions (unless of course the old ones are rebilded) according to
https://github.com/arendst/Tasmota/commit/98cbf2587a1a914bbd16996ebb48dd451d3da448

open source software is therefore called so, because it is available (openly) to everyone without restrictions.

Probably now it is fashionable to build an "iron curtain" in front of the Russian Federation. But by such actions you describe yourself as a developer.

Who will trust the software (especially for embedded devices that are connected to electricity) that can stop working/become unusable/cause a fire/ because something has gone wrong in the world.

What's next? is there a bomb? attempts to cause epilepsy by blinking light? restrictions for everyone who doesn't play Cold War 2.0?

Who now wants to put a compiled binary file if there are similar traps there.

Please don't do this.

**Comment by arendst** [comment]
Thanks for your concerns.

Lucky for the world this Open Source project is still available for the whole world including the Russian Federation. Nothing to worry about possible malfunctions.

I'm not sure what triggered you to open this discussion but if it was because of the additional text banner then I have reached my goal.

Pls ask yourselves why you react this way and if possible think about the consequences of your leader(s) actions. Currently cities are destoyed. citizens like you and me are murdered for no reason other than mis-communication. 

![image](https://user-images.githubusercontent.com/11044339/156577165-d3447857-9b11-4693-bdb9-611a19696f79.png)

> A verified image first acquired by [@bellingcat](https://twitter.com/bellingcat) shows the attack included Russian-manufactured 300mm Smerch rockets with 9N235 cluster munitions. The 2019 manufacture date, after Russia stopped selling these arms to Ukraine, indicates that the attack was launched by Russia.
> 
> Amnesty International
> 13:14 - 3 maart 2022
> 

To ease your concerns I suggest to scan the code and notice what causes the message and what damage will (not) be done.

I wish you all great wisdom in the near future.

Theo Arends.

**Reply by mikesrus** [reply]
Oh, war never happens from scratch. Do you really think that everything happened in 2022? No, it happened in 2014.

Do you know about this https://en.wikipedia.org/wiki/Azov_Battalion
https://en.wikipedia.org/wiki/2014_Odessa_clashes
https://cdni.rt.com/russian/images/7/2/d/72dd2949b95f2a9b8a04d675bbff37586d1c5ed6_article.jpg

Alas, there are wars and they go on all the time. half of the world is burning now anyway. And many EU countries participate in them.
Doesn't Bellingcat tell you about it?


But I'm talking about something else. there are many resources for expressing your own opinion. the same reddit facebook pikabu or you can just talk to random people in order to form an opinion.

It is not necessary to mix work emotions and politics. As a result, distrust and decline are born.

If you have a desire to hear a different point of view in order to be a little more objective in your views, you can always contact people. at least with me, for example.

it is not necessary to assume that the Russians are delighted with what is happening, but it took a very long time to get to these events.

**Reply by Bigmanekb** [reply]
> 

Official statement of Russian Foreign Department about crimes in Donbass in 2014 - 2022 

https://cloud.mail.ru/public/ahqT/mEV7SWjAK

If you want know the truth, listen to the both sides.

**Reply by Bigmanekb** [reply]
Learn more... https://ukrainian-crimes.org

**Comment by joba-1** [comment]
My opinion on this

It is still open source. Feel free to fork and develop to your liking.

Since it is open source you can still trust it or do it like Lenin said: trust is good, control is better.

The changes seem to be of symbolic nature and maybe a minor annoyance. Nothing to compare what ukrainian people have to endure now. I have no objections if my devices start behaving like this.

Maybe it helps to make people that might be ill informed right now aware of the situation so they may seek more independent information and then act accordingly.

Please don’t get me wrong: I know some russians, so I know first hand they can be nice just like anyone else in the world. Unfortunately everywhere are the good ones and the idiots - fortunately our Schröder isn‘t active in politics anymore.

Think,
Стоп войну

**Comment by TeiSinTai** [comment]
I find it amusingly appropriate using language id targeting in support of Ukraine. Otherwise, I wonder, what are the limits for applying politics in open source? What is and isn't appropriate? How many commits from russian-speeaking authors should people in NATO countries check for additional functionality from now on, for example?

**Reply by arendst** [reply]
At first I only used the language id but then I was addressed by Kasachstan who felt they were not to blame.

So in addition I started using GEO positioning based on the latitude/longitude needed to calculate sunrise/sunset.

For me it was a nice excerise. I'll keep it in the repository but I disabled it an hour ago.

So friends again ;-)

**Reply by TeiSinTai** [reply]
Hm. Let me expand upon my point of amusement. It is a part of russian official statement that there are russian-speaking people discriminated out there, many of them in Ukraine. So, you using russian language as a part of discriminator works in support of that position, proving them right. Nice work! =)

**Reply by joba-1** [reply]
I am not so amused about someone comparing looking at a message on one side with being killed on the other side - quite a different category of discrimination, more like one is and one isn't, don't you think?

And war propaganda never needs support or reason. Just one example:
The reason given for starting the "special operation" war was that Ukraine needs to stay neutral for Russias security.
I would never think it is justified if I start a war against my neighbour simply because he does not do what I want in his territory. 
Why should he? And in what way is starting a war more secure than anything else? It just doesn't make any sense.

**Reply by mikesrus** [reply]
If we have reached an understanding and consensus, then let's get it over with.
There are other resources for politics.

**Reply by DeeKey** [reply]
> I would never think it is justified if I start a war against my neighbour simply because he does not do what I want in his territory.

Well, that was exactly the case of this issue:
https://en.wikipedia.org/wiki/Cuban_Missile_Crisis

**Reply by joba-1** [reply]
> Well, that was exactly the case of this issue:

Ad hominem, logical fallacy, tu quoque, two wrongs, ...
